scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -340,7 +365,7 @@ trait SequenceFileScheme extends SchemedSource {
  */
 trait SuccessFileSource extends FileSource {
   override protected def pathIsGood(p: String, conf: Configuration) =
-    FileSource.globHasNonHiddenPaths(p, conf) && FileSource.globHasSuccessFile(p, conf)
+    FileSource.allPathsCommitted(p, conf)
```

Comment:
This change in behavior should probably be in its own PR and more flagged for discussion I think as to how it should be ANN. Requiring more successfiles seems very safe, dropping the non-hidden paths 'weakens' the test (even if its maybe more correct). Probably just deserves its own flagging/PR to see if it should be in this trait or a new one.